tests/phpunit/Persistence/SitelinkBasedStatementsLookupTest.php

```diff
@@ -41,18 +41,18 @@ protected function setUp(): void {
 		);
 	}
 
-	public function testPageWithoutSitelinkReturnsNoStatements(): void {
```

Comment:
This name is misleading in two ways:
* The page is not returning anything, it's the lookup under test that is
* The page has no sitelinks. It's being referenced by a sitelink (or not in this case)